src/features/PassSurvey/ui/SurveyHeader.tsx

```diff
@@ -25,6 +26,12 @@ const SurveyHeader = (props: Props) => {
     return str.length > length ? `${str.substring(0, length)}...` : str;
   };
 
+  const title = props.title ?? context.activity.name;
```

Comment:
I wonder why it is supposed to be like this. When is the `context.activity.name` case used?

Reply:
Most of the time the title is used from the context.

Reply:
The props title used only when you want to provide some specific title